src/components/Gauge.js

```diff
@@ -1,53 +1,30 @@
 import * as React from 'react'
-const Pie = require('paths-js/pie')
 import styled, { withTheme } from 'styled-components'
-const map = require('lodash/map')
+import { map } from 'lodash'
 import { compose } from 'recompose'
-import { px } from '../utils'
+import { Chart } from 'chart.js'
 
 import asRendition from '../asRendition'
 import theme from '../theme'
+import { doughnutLabel, doughnutDefaults } from '../chartPlugins'
 
 const { font } = theme
 
 const StatusGaugeWrapper = styled.div`
   display: block;
   color: ${({ theme }) => theme.colors.text.main};
 
-  /* fix edge hover missplacing bug
-    on svg text inside anchors
-  */
-  &,
-  &:hover {
-    text-decoration: none;
-
-    text {
-      text-decoration: none !important;
-    }
-
-    .gauge__graph__total-label {
-      transform: translateX(0);
-    }
-  }
-
-  .gauge__graph {
+  .gauge__doughnut {
```

Comment:
Could this still be `.gauge__graph` since we [are already using this in the UI to add some margins](https://github.com/resin-io/resin-ui/blob/26868a2ecb592025e4922c2aad8ef39d3841073b/src/scripts/modules/applications-list/applications-list.less#L27).
We should confirm whether this would affect the UI in a negative way.